Bugsnag/KSCrash/Source/KSCrash/Recording/BSG_KSCrashNames.c

```diff
@@ -0,0 +1,28 @@
+//
+//  BSG_KSCrashNames.c
+//  Bugsnag
+//
+//  Created by Karl Stenerud on 28.09.21.
+//  Copyright © 2021 Bugsnag Inc. All rights reserved.
+//
+
+#include "BSG_KSCrashNames.h"
+#include <mach/thread_info.h>
+
+static const char* thread_state_names[] = {
+    // Defined in mach/thread_info.h
+    NULL,
+    "running", // TH_STATE_RUNNING = 1
+    "stopped", // TH_STATE_STOPPED = 2
+    "waiting", // TH_STATE_WAITING = 3
+    "uninterruptible", // TH_STATE_UNINTERRUPTIBLE = 4
+    "halted", // TH_STATE_HALTED = 5
```

Comment:
Would it be better to use same strings as the consts in `mach/thread_info.h`? They still convey the same meaning but I think make it more clear where they came from.